### PR TITLE
[Copy] Update auth terms to match GCKey

### LINKED ISSUE #5367: ⌨  Sign in and Sign up instead of Login and Register (to match GCKey)
## Insight/Issue:
Inconsistent terms for account creation between GC Digital Talent and GC Key (“register” vs “Sign up”). Also should change Login to sign in to match GC Key. 
### Location:
All GC Digital Talent pages
### Category:
Service design/Copy Issue 
## Reproduction info:

1. Access the Home page with a screen reader (narrator/NVDA)and listen as you go through. 
2. Activate the "Register" button/link
3. Activate "Continue to GC Key and Register" link
4. Find where to register on GC Key using the available instructions.

### Fable Video Clip:
[Register on GC Digital Talent vs sign up on GC Key.](https://app.makeitfable.com/clip/63c32621702a95001a7ca297)
## Expected Behaviour:
Consistent terms for all users in English and French. Consistent terms for all users in English and French. En: Sign-in /Sign up Fr: Se connecter/S’enregistrer
### Notes:
Register links appears in the header of all pages across the platform. However,  when users are ushered to the GC Key portal, they are asked to sign up for a GC Key. Though register is clearer, it adds confusion for screen reader users once they arrive on the GC Key Welcome page and are directed to a sign in component and then must tab through to find a way to “sign up” for a GC Key account.
### Potential WCAG Considerations (if any):
1.3.1 Info and Relationships, 1.3.2 Meaningful Sequence
### Acceptance Criteria: 

- "Log in" is changed to "Sign in" everywhere on the platform
- "Register" is changed to "Sign up" everywhere on the platform
- "Connexion" is changed to "Se connecter" everywhere on the platform
- Change "S'inscrire" to "S'enregistrer" everywhere on the platform
- no URLs to be changed

### PR DESCRIPTION
**Pull request by esizer**
🤖 Resolves #5367 

## 👋 Introduction

This updates the following terms and variations of them:

- Login -> Sign In
- Logout -> Sign Out
- Register -> Sign Up

## 🧪 Testing

Assist reviewers with steps they can take to test that the PR does what it says it does.

1. Build `npm run dev`
2. Confirm instances of the old terms no longer exist (except in URLs)
3. Search the app for those terms confirming they no longer exist

## 📸 Screenshot

Screenshot 2023-07-18 103948
Screenshot 2023-07-18 103956

### CODE REVIEW

**Comment by vd1992** [comment]
The e2e is failing, looks like a few assertions still referencing deleted text

**Comment by esizer** [comment]
> The e2e is failing, looks like a few assertions still referencing deleted text

Hmm, I swear I fixed this but I guess there are some more that I missed 😢

File: apps/web/src/pages/Auth/SignInPage/SignInPage.tsx [hunk 1]
```diff
@@ -107,17 +101,17 @@ const LoginPage = () => {
               <p>
                 {intl.formatMessage({
                   defaultMessage:
-                    "You can log into your IT Apprenticeship Program for Indigenous Peoples profile using your existing GCKey, even if you’ve never used this platform before.",
-                  id: "ySpJ2L",
+                    "You can log into your IT Apprenticeship Program for Indigenous Peoples profile using your existing GCKey, even if you've never used this platform before.",
+                  id: "0UeJoO",
```
Comment (vd1992) [inline apps/web/src/pages/Auth/SignInPage/SignInPage.tsx:105]: Need to change the "log into" right?

File: apps/web/src/pages/Auth/CreateAccountPage/CreateAccountPage.tsx [hunk 2]
```diff
@@ -132,19 +132,19 @@ export const CreateAccountForm = ({
...
-                  "Title for successful login alert in create account page.",
+                  "Title for successful sign in alert in create account page.",
               })}
             </Alert.Title>
             <p>
               {intl.formatMessage({
                 defaultMessage:
                   "Welcome to the GC Digital Talent platform. Moving forward, you can log into your profile using the same GCKey username and password.",
-                id: "cmZ9cx",
+                id: "HHIXc0",
```
Comment (vd1992) [inline apps/web/src/pages/Auth/CreateAccountPage/CreateAccountPage.tsx:145]: "log into"

**Review by vd1992 — CHANGES_REQUESTED** [review]
A few instances of "log into" present. The commented as well as this one
Two such cases in the file
`apps/web/src/pages/ProfileAndApplicationsPage/components/ProfileAndApplicationsHeading.tsx`

image

File: apps/web/src/pages/Auth/CreateAccountPage/CreateAccountPage.tsx (hunk 2 above)
Comment (esizer) [reply apps/web/src/pages/Auth/CreateAccountPage/CreateAccountPage.tsx]: Fixed in [fix straggling login strings](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7303/commits/1b5b97498ef0fa832d41f02c9c846384038eb67f)

File: apps/web/src/lang/fr.json [hunk 3]
```diff
@@ -1210,6 +1214,14 @@
     "defaultMessage": "Voir les candidats<hidden> pour {label}</hidden>",
     "description": "Text for a link to the Pool Candidates table"
   },
+  "6SfJEp": {
+    "defaultMessage": "You can sign in to your IT Apprenticeship Program for Indigenous Peoples profile using your existing GCKey, even if you've never used this platform before.",
+    "description": "Instructions on how to sign in with GCKey - IAP variant"
+  },
```
Comment (vd1992) [inline apps/web/src/lang/fr.json:1220]: Uh, not sure this should be here in `fr.json`

File: apps/web/src/pages/Auth/SignInPage/SignInPage.tsx [hunk 4]
```diff
@@ -71,16 +65,16 @@ const LoginPage = () => {
               <p>
                 {intl.formatMessage({
                   defaultMessage:
-                    "You can log into your GC Digital Talent profile using your existing GCKey, even if you've never used this platform before.",
-                  id: "NcB+TT",
-                  description: "Instructions on how to login with GCKey",
+                    "You can sign into your GC Digital Talent profile using your existing GCKey, even if you've never used this platform before.",
```
Comment (mnigh) [inline apps/web/src/pages/Auth/SignInPage/SignInPage.tsx:68]: Replace sign into with sign in to

File: apps/web/src/pages/Auth/SignUpPage/SignUpPage.tsx [hunk 5]
```diff
@@ -72,16 +66,16 @@ const RegisterPage = () => {
               <p>
                 {intl.formatMessage({
                   defaultMessage:
-                    "You can log into your GC Digital Talent profile using your existing GCKey, even if you've never used this platform before.",
-                  id: "NcB+TT",
-                  description: "Instructions on how to login with GCKey",
+                    "You can sign into your GC Digital Talent profile using your existing GCKey, even if you've never used this platform before.",
```
Comment (mnigh) [inline apps/web/src/pages/Auth/SignUpPage/SignUpPage.tsx:69]: Replace sign into with sign in to

File: apps/web/src/pages/Auth/SignUpPage/SignUpPage.tsx [hunk 6]
```diff
@@ -118,8 +112,8 @@ const RegisterPage = () => {
                 {intl.formatMessage(
                   {
                     defaultMessage:
-                      "GCKey is a service that allows you to log into <a>many different government services</a> with a single username and password. The button below will take you to GCKey’s website, where you’ll be able to create an account if you don’t already have one. Once you’ve registered for GCKey, you’ll be brought back here to complete your profile.",
-                    id: "cPgH94",
+                      "GCKey is a service that allows you to sign into <a>many different government services</a> with a single username and password. The button below will take you to GCKey's website, where you'll be able to create an account if you don't already have one. Once you've signed up for GCKey, you'll be brought back here to complete your profile.",
```
Comment (mnigh) [inline apps/web/src/pages/Auth/SignUpPage/SignUpPage.tsx:115]: Replace sign into with sign in to

File: apps/web/src/pages/ProfileAndApplicationsPage/components/ProfileAndApplicationsHeading.tsx [hunk 7]
```diff
@@ -194,8 +194,8 @@ const DashboardHeading = ({ user }: DashboardHeadingProps) => {
             {intl.formatMessage(
               {
                 defaultMessage:
-                  "When you log into your account, you'll start on this page from now on. You can use the <strong><a>Track your applications</a></strong> section on this page to review your application to the IT Apprenticeship Program for Indigenous Peoples and track your position in the program.",
-                id: "Z9wBwT",
+                  "When you sign into your account, you'll start on this page from now on. You can use the <strong><a>Track your applications</a></strong> section on this page to review your application to the IT Apprenticeship Program for Indigenous Peoples and track your position in the program.",
```
Comment (mnigh) [inline apps/web/src/pages/ProfileAndApplicationsPage/components/ProfileAndApplicationsHeading.tsx:197]: Replace sign into with sign in to

**Review by mnigh — COMMENTED** [review]
Found a few more **sign into** references.

File: apps/web/src/lang/fr.json [hunk 8]
```diff
@@ -388,6 +388,10 @@
     "defaultMessage": "De la même manière que vous avez sélectionné des éléments de votre parcours professionnel pour confirmer les exigences en matière d’expérience et d’études, nous vous demanderons de décrire une ou plusieurs expériences de votre parcours professionnel au cours desquelles vous avez activement utilisé la compétence requise.",
     "description": "Application step for skill requirements, introduction, description, paragraph two"
   },
+  "0OLsay": {
+    "defaultMessage": "CléGC est un service qui vous permet de vous connecter à <a> de nombreux services gouvernementaux différents</a> avec un seul nom d’utilisateur et un seul mot de passe. Le bouton ci-dessous vous mènera au site Web de CléGC, où vous pourrez créer un compte si vous n’en avez pas encore un. Une fois que vous vous serez inscrit(e) à CléGC, vous reviendrez ici pour compléter votre profil.",
```
Comment (mnigh) [inline apps/web/src/lang/fr.json:392]:
```suggestion
    "defaultMessage": "CléGC est un service qui vous permet de vous connecter à <a>de nombreux services gouvernementaux différents</a> avec un seul nom d’utilisateur et un seul mot de passe. Le bouton ci-dessous vous mènera au site Web de CléGC, où vous pourrez créer un compte si vous n’en avez pas encore un. Une fois que vous vous serez inscrit(e) à CléGC, vous reviendrez ici pour compléter votre profil.",
```

File: apps/web/src/lang/fr.json [hunk 9]
```diff
@@ -2593,6 +2617,10 @@
     "defaultMessage": "J'envisagerais d'accepter un emploi :",
     "description": "Legend Text for required work preferences options in work preferences form"
   },
+  "GQsJAi": {
+    "defaultMessage": "Lorsque vous vous connectez à votre compte, vous commencez par cette page. Vous pouvez utiliser l'outil <strong><a>pour suivre vos candidatures </a></strong> sur cette page afin de revoir votre demande au Programme d'apprentissage en TI pour les personnes autochtones et de suivre votre statut dans le Programme.",
```
Comment (mnigh) [inline apps/web/src/lang/fr.json:2621]:
```suggestion
    "defaultMessage": "Lorsque vous vous connectez à votre compte, vous commencez par cette page. Vous pouvez utiliser l'outil <strong><a>pour suivre vos candidatures</a></strong> sur cette page afin de revoir votre demande au Programme d'apprentissage en TI pour les personnes autochtones et de suivre votre statut dans le Programme.",
```

File: apps/web/src/lang/fr.json [hunk 10]
```diff
@@ -2750,6 +2782,10 @@
     "defaultMessage": "(À déterminer)",
     "description": "Message displayed when a pool has no expiry date yet"
   },
+  "Hf3x3E": {
+    "defaultMessage": "Lorsque vous vous connectez à votre compte, vous commencez par cette page. Vous pouvez utiliser l'outil <strong><a>pour suivre vos candidatures </a></strong> sur cette page afin de revoir votre demande au Programme d'apprentissage en TI pour les personnes autochtones et de suivre votre statut dans le Programme.",
```
Comment (mnigh) [inline apps/web/src/lang/fr.json:2786]:
```suggestion
    "defaultMessage": "Lorsque vous vous connectez à votre compte, vous commencez par cette page. Vous pouvez utiliser l'outil <strong><a>pour suivre vos candidatures</a></strong> sur cette page afin de revoir votre demande au Programme d'apprentissage en TI pour les personnes autochtones et de suivre votre statut dans le Programme.",
```

File: apps/web/src/pages/Auth/SignUpPage/SignUpPage.tsx [hunk 11]
```diff
@@ -118,8 +112,8 @@ const RegisterPage = () => {
                 {intl.formatMessage(
                   {
                     defaultMessage:
-                      "GCKey is a service that allows you to log into <a>many different government services</a> with a single username and password. The button below will take you to GCKey’s website, where you’ll be able to create an account if you don’t already have one. Once you’ve registered for GCKey, you’ll be brought back here to complete your profile.",
-                    id: "cPgH94",
+                      "GCKey is a service that allows you to sign in to <a>many different government services</a> with a single username and password. The button below will take you to GCKey's website, where you'll be able to create an account if you don't already have one. Once you've signed up for GCKey, you'll be brought back here to complete your profile.",
```
Comment (mnigh) [inline apps/web/src/pages/Auth/SignUpPage/SignUpPage.tsx:115]: "button below" is using directional language.

Comment (mnigh) [reply apps/web/src/pages/Auth/SignUpPage/SignUpPage.tsx:115]: Out of scope, [issue](https://github.com/GCTC-NTGC/gc-digital-talent/issues/7375) created.

**Review by mnigh — COMMENTED** [review]
Looks pretty good. A few small things: was there a reason you renamed some components and not others?

## Not renamed

- `LoginOrLogout`
- `LogoutConfirmation`

## Renamed

- `LoginPage`
- `RegisterPage`

Also, found this:

https://github.com/GCTC-NTGC/gc-digital-talent/blob/79ee3c67208d2fe60a3a4f6d15d9850b38832dbb/packages/auth/src/components/RequireAuth.tsx#L63

**Review by vd1992 — APPROVED** [review]
My concerns are all addressed, not entirely sure if Matt has anything else to add.
🏆 

An aside, not new to this PR, but I still find it confusing all three of these links point to the same place. 

image

**Comment by esizer** [comment]
> Looks pretty good. A few small things: was there a reason you renamed some components and not others?

Honestly, just missed them 🤭 Thanks for catching that. Renamed in [rename components to match new auth terms](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7303/commits/084a348e0cea8c4bfe86565fdb6ffb18c9d1fecb)

**Review by mnigh — APPROVED** [review]
👍

**Pull request merged**